app/models/validations/household_validations.rb

```diff
@@ -19,6 +19,11 @@ def validate_reason_for_leaving_last_settled_home(record)
       record.errors.add :reason, I18n.t("validations.household.underoccupation_benefitcap.dont_know_required")
     end
     validate_other_field(record, 31, :reason, :other_reason_for_leaving_last_settled_home)
+
+    if record.reason == 1 && record.referral.present? && !record.is_internal_transfer?
```

Comment:
Worth a wrapper method to help make it obvious what 1 is?

Reply:
Yo, you speedy! Will do!